mass-deployer/example/conf.yaml

```diff
@@ -0,0 +1,33 @@
+node_groups:
+  - name: group_a
+    nodes_count: 3 # amount of nodes to be found
+    free_cpu: 2 # number of logical cores
+    free_mru: 16384 # amount of memory in mb
+    free_ssd: 100 # amount of ssd storage in gb
+    free_hdd: 50 # amount of hdd storage in gb
+    dedicated: false # is dedicated nodes?
+    pubip4: false # should the nodes have free ip v4
+    pubip6: false # should the nodes have free ip v6
+    certified: false # should the nodes be certified(if false the nodes could be certified of diyed) 
+    region: "europe" # region could be the name of the continents the nodes are located in (eg: europe, asia,...)
+    min_bandwidth_ms: 100
```

Comment:
What is `min_bandwidth_ms`?

Reply:
It's not implemented yet

Reply:
We can remove it as it is not supported and create an issue for it.